automation/vars/main.yml

```
@@ -366,12 +370,18 @@ pgbouncer_auth_user: true # or 'false' if you want to manage the list of users f
 pgbouncer_auth_username: pgbouncer # user who can query the database via the user_search function
 pgbouncer_auth_password: "" # If not defined, a password will be generated automatically during deployment
 pgbouncer_auth_dbname: "postgres"
-pgbouncer_client_tls_sslmode: "disable"
-pgbouncer_client_tls_key_file: ""
-pgbouncer_client_tls_cert_file: ""
+pgbouncer_client_tls_sslmode: "require"
+pgbouncer_client_tls_key_file: "{{ pgbouncer_conf_dir }}/server.key"
+pgbouncer_client_tls_cert_file: "{{ pgbouncer_conf_dir }}/server.crt"
```

```suggestion
pgbouncer_client_tls_key_file: "{{ postgresql_conf_dir }}/server.key"
pgbouncer_client_tls_cert_file: "{{ postgresql_conf_dir }}/server.crt"
```

We can use the Postgres key and certificate.

---

I set the path of variable pgbouncer_conf_dir since PGBouncer role executes before Patroni role